packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -202,6 +202,9 @@ export class SigningCosmWasmClient extends CosmWasmClient {
   private readonly signer: OfflineSigner;
   private readonly aminoTypes: AminoTypes;
   private readonly gasPrice: GasPrice | undefined;
+  // Starting with Cosmos SDK 0.47, we see many cases in which 1.3 is not enough anymore
+  // E.g. https://github.com/cosmos/cosmos-sdk/issues/16020
+  private readonly gasMultiplier = 1.4;
```

Comment:
Nice, can you call this `defaultGasMultiplier` since it is only used when the user does not specify anything